scripts/styleChecker.fsx

```diff
@@ -0,0 +1,393 @@
+            Process
+                .Execute(
+                    {
+                        Command = "dotnet"
+                        Arguments = "tool list"
+                    },
+                    Echo.Off
+                )
+                .UnwrapDefault()
+
+        installedPackages.Split Environment.NewLine
+        |> Seq.map(fun line ->
+            line.Contains "fantomless-tool"
+            && line.Contains fantomlessToolVersion
+        )
+        |> Seq.contains true
+
+    if not(isFantomlessInstalled) then
+        Process
+            .Execute(
+                {
+                    Command = "dotnet"
+                    Arguments = "new tool-manifest --force"
+                },
+                Echo.Off
+            )
+            .UnwrapDefault()
+        |> ignore
+
+        Process
+            .Execute(
+                {
+                    Command = "dotnet"
+                    Arguments =
+                        $"tool install fantomless-tool --version {version}"
+                },
+                Echo.Off
+            )
+            .UnwrapDefault()
+        |> ignore
+
+let UnwrapProcessResult
+    (suggestion: string)
+    (raiseError: bool)
+    (processResult: ProcessResult)
+    : string =
+    let errMsg =
+        sprintf
+            "Error when running '%s %s'"
+            processResult.Details.Command
+            processResult.Details.Args
+
+    match processResult.Result with
+    | Success output ->
+        Console.WriteLine output
+        output
+    | Error(_, output) ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = errMsg + Environment.NewLine + suggestion
+        Console.Error.WriteLine fullErrMsg
+
+        if raiseError then
+            raise <| ProcessFailed errMsg
+        else
+            fullErrMsg
+    | WarningsOrAmbiguous output ->
+        if processResult.Details.Echo = Echo.Off then
+            output.PrintToConsole()
+            Console.WriteLine()
+            Console.Out.Flush()
+
+        let fullErrMsg = sprintf "%s (with warnings?)" errMsg
+        Console.Error.WriteLine fullErrMsg
+        fullErrMsg
+
+let IsProcessSuccessful(processResult: ProcessResult) : bool =
+    match processResult.Result with
+    | Success output -> true
+    | _ -> false
+
+let InstallPrettier(version: string) =
+    let isPrettierInstalled =
+        Process.Execute(
+            {
+                Command = "npm"
+                Arguments = $"list prettier@{version}"
+            },
+            Echo.All
+        )
+        |> IsProcessSuccessful
+
+    if not(isPrettierInstalled) then
+        Process.Execute(
+            {
+                Command = "npm"
+                Arguments = $"install prettier@{version}"
+            },
+            Echo.Off
+        )
+        |> UnwrapProcessResult "" true
```

Comment:
@realmarv whenever you are writing `""`, please please please:
- Either write `String.Empty`, or:
- Use `None` (in this case None is better).

Reply:
Done